components/navigation/navigation.css

```diff
@@ -0,0 +1,60 @@
+.navbar-container {
+    display: flex;
+    background-color: #cde9e6;
+    height: 3.5em;
+    padding: 0.5em 1em;
+    justify-content: space-between;
+    font-size: 1em;
+    align-items: center;
+    text-decoration: none;
+}
+.navbar-link {
+    display: flex;
+}
+.brand-logo {
+    margin-left: -2em;
+}
+.nav-item {
+    list-style-type: none;
+    padding: 0 0.6em;
+    text-decoration: none;
+}
+a {
+    text-decoration: none;
+}
+.navbar-brand, .nav-link{
+    color: #2a9d8f;
```

Comment:
you are using this color more then one place you should use color variables instead i think.